src/main/resources/sponsors.json

```diff
@@ -0,0 +1,19 @@
+[{
```

Comment:
Format json file.
Please format all the file in project usong `Ctrl + Alt + Shift + L`.